CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Pipeline updates
 
+* Add option to modify `featureCounts -t` option to assign reads to a different type than `exon`. This makes it compatible with nuclearRNAseq.
```

Comment:
counting reads in introns and not only in exons if you specify `gene` for that option and not exon as it is the default. I can add it there.

Reply:
We like it because we’re a sequencing platform who send off the BAM files to users. This header embeds where it came from for post-publication analysis etc.

Reply:
done. let me know if that is enough.